src/udev/net/link-config.c

```diff
@@ -1314,6 +1362,52 @@ int config_parse_wol_password(
         return 0;
 }
 
+int config_parse_rps_cpu_mask(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        LinkConfig *config = ASSERT_PTR(userdata);
+        CPUSet *mask = NULL;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (!config->rps_cpu_mask) {
+                mask = new0(CPUSet, 1);
+                if (!mask)
+                        return log_oom();
+
+                config->rps_cpu_mask = mask;
+        }
```

Comment:
If `config->rps_cpu_mask` is newly allocated here, then free it when the parser below failed.
Otherwise, specifying an invalid string is equivalent to `"reset"` or so suggested in the above comment.

Reply:
Ok, I will free it, then I think an invalid string will be equivalent to 'nothing specified'.